boards/adafruit-feather-rp2040/README.md

```diff
@@ -16,7 +16,7 @@ RP2040 chip according to how it is connected up on the Feather.
 To use this crate, your `Cargo.toml` file should contain:
 
 ```toml
-adafruit-feather-rp2040 = "0.2.0"
+adafruit-feather-rp2040 = "0.3.0"
```

Comment:
Could be:
```suggestion
adafruit-feather-rp2040 = "0.3.1"
```
(Doesn't matter much due to semver.)

Reply:
Yeah. I was relying on sem ver for these
Think I should change it to two figures (eg "0.3" to make this intention more clear?
We don't want to encourage PRs bumping just the patch version

Reply:
Going to merge without making a decision about this since I managed to break examples on main (by deleting the 0.5.0 branches prematurely)

Reply:
Sure, it was just a minor nitpick, no need to delay the merge.